src/java/org/apache/cassandra/cql3/statements/schema/CreateTableStatement.java

```diff
@@ -412,16 +412,6 @@ public Set<String> clientWarnings(KeyspacesDiff diff)
     {
         ImmutableSet.Builder<String> warnings = ImmutableSet.builder();
 
-        int tableCount = Schema.instance.getNumberOfTables();
```

Comment:
Rather than removing this block should it be changed to check the guardrail?  I don't see any reference to the guardrail in the client warnings function here.

Reply:
I think that's because guardrails are implemented in `validate` rather than `clientWarnings` - I don't know why that is, but it's the same in OSS C* afaict.